hash-tables/ex2/ex2.py

```diff
@@ -20,4 +20,14 @@ def reconstruct_trip(tickets, length):
     YOUR CODE HERE
     """
 
+    for ticket in tickets:
+        hash_table_insert(hashtable, ticket.source, ticket.destination)
+
```

Comment:
Great job again with your variable naming Bryce 👍 , when reading someone else's code clear naming you've got here goes a long way in understanding it more quickly, so it's a great habit you've developed.